mlir/include/mlir/Dialect/Rock/IR/Rock.h

```diff
@@ -58,6 +58,9 @@ namespace rock {
 ArrayAttr noTransformsArray(Builder &b, size_t n);
 
 ArrayAttr getIndexArrayAttr(Builder &b, ArrayRef<int64_t> values);
+
+const char reverseGridAttrName[] = "reverse_grid";
```

Comment:
1. Upstream merge'll make this nicer
2. Put this in the tablegen file with all the other `StringLiteral`s